### PR TITLE
Fix importing from JSON file where some property values are null

### LINKED ISSUE #3737: Unable to import an object from JSON
Export an object from the navigation tree
Right click on the My Items folder and choose Import from JSON
Select the file exported in Step 1 and press OK

Expected: The new object is created from the exported JSON
Actual: Error: 
```
Uncaught (in promise) TypeError: domainObject is undefined
    check webpack-internal:///./src/plugins/objectMigration/Migrations.js:182
    needsMigration webpack-internal:///./src/plugins/objectMigration/plugin.js:33
    needsMigration webpack-internal:///./src/plugins/objectMigration/plugin.js:33
    migrate webpack-internal:///./src/plugins/objectMigration/plugin.js:45
    promise callback*migrate webpack-internal:///./src/plugins/objectMigration/plugin.js:44
    load webpack-internal:///./src/api/composition/CompositionCollection.js:223
    load webpack-internal:///./src/api/composition/CompositionCollection.js:223
    promise callback*__WEBPACK_AMD_DEFINE_RESULT__</CompositionCollection.prototype.load webpack-internal:///./src/api/composition/CompositionCollection.js:222
    invoke webpack-internal:///./src/adapter/capabilities/AlternateCompositionCapability.js:94
    then webpack-internal:///./platform/core/src/capabilities/MutationCapability.js:49
    add webpack-internal:///./src/adapter/capabilities/AlternateCompositionCapability.js:59
    importObjectTree webpack-internal:///./platform/import-export/src/actions/ImportAsJSONAction.js:80
    perform webpack-internal:///./platform/import-export/src/actions/ImportAsJSONAction.js:54
    Angular 17
    populateElement webpack-internal:///./platform/representation/src/TemplateLinker.js:123
    showTemplate webpack-internal:///./platform/representation/src/TemplateLinker.js:128
    changeTemplate webpack-internal:///./platform/representation/src/TemplateLinker.js:161
    link webpack-internal:///./platform/representation/src/TemplateLinker.js:167
```

### PR DESCRIPTION
**Pull request by shefalijoshi**
If value is null, don't process it.
Resolves #3737 

### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? N/A
3. Command line build passes? Y
4. Changes have been smoke-tested? Y
5. Testing instructions included? Y

### CODE REVIEW

File: platform/import-export/src/actions/ImportAsJSONAction.js [hunk 1]
```diff
@@ -154,7 +154,8 @@ define(['zepto', 'objectUtils'], function ($, objectUtils) {
         tree = JSON.stringify(tree).replace(new RegExp(oldIdKeyString, 'g'), newIdKeyString);
 
         return JSON.parse(tree, (key, value) => {
-            if (Object.prototype.hasOwnProperty.call(value, 'key')
+            if (value !== null
```
Comment (nikhilmandlik) [inline platform/import-export/src/actions/ImportAsJSONAction.js:157]: how about `undefined`?

Comment (shefalijoshi) [reply platform/import-export/src/actions/ImportAsJSONAction.js:157]: Do you mean I should add value !== undefined as well?

Comment (shefalijoshi) [reply platform/import-export/src/actions/ImportAsJSONAction.js]: Done.

**Review by nikhilmandlik — APPROVED** [review]
LGTM

**Comment by nikhilmandlik** [comment]
### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue? | Y
Appropriate unit tests included? | NA
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (only applicable if this PR closes the issue) | Y

**Pull request merged**